communication guidelines.md

```diff
@@ -0,0 +1,76 @@
+## Purpose:
+[GitHub Actions](https://docs.github.com/en/actions/learn-github-actions) gives you the flexibility to build an automated software development lifecycle workflow. With [GitHub Actions for Azure](https://docs.microsoft.com/en-gb/azure/developer/github/) you can create workflows that you can set up in your repository to build, test, package, release and deploy to Azure
+This document is to set communication guidelines for making GitHub Actions ready for the external community. We want to set a robust framework for communication, and governance for anything related to GitHib Actions for Azure, for enabling both internal and external users to be always updated on any change in this area.
+
+## Ground Rules for Communication:
+1. ANy form of communciation should abide by the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/)
+2. We would use GitHub aliases for all the communication
+3. We would want all communication to be public. In case of communication that involves strategic decisions which partner team would not want to go public earlier official announcement, have all the communication and key decisions in the form of GH Discussions/Issues inside the private repo. Once right time arises, the repo would be made public
+
+## What should be communicated:
```

Comment:
Should we mention this one line here to emphasise that this list is not exhaustive
"As mentioned in the rules for communication, we want all the communication to be public(with only some exceptions in the next section). This includes the below things -

Reply:
Removed the exceptions from this doc, since this is for the whole community to read and follow. On a second thought I felt exceptions can be discussed on case to case basis on slack/emails and decisions can be taken based on conversation.